README.md

```diff
@@ -1,2 +1,149 @@
+  "sec-fetch-mode": "navigate",
+  "sec-fetch-user": "?1",
+  "sec-fetch-dest": "document"
+}
+```
+## API Reference
+All public classes, methods and their parameters can be inspected in this API reference.
+
+<a name="HeaderGenerator"></a>
+
+### HeaderGenerator
+HeaderGenerator randomly generates realistic browser headers based on specified options.
+
+
+* [HeaderGenerator](#HeaderGenerator)
+    * [`new HeaderGenerator(options)`](#new_HeaderGenerator_new)
+    * [`.getHeaders(options)`](#HeaderGenerator+getHeaders)
+
+
+* * *
+
+<a name="new_HeaderGenerator_new"></a>
+
+#### `new HeaderGenerator(options)`
+
+| Param | Type | Description |
+| --- | --- | --- |
+| options | [<code>HeaderGeneratorOptions</code>](#HeaderGeneratorOptions) | default header generation options used unless overridden |
+
+
+* * *
+
+<a name="HeaderGenerator+getHeaders"></a>
+
+#### `headerGenerator.getHeaders(options)`
+Generates a single set of headers using a combination of the default options specified in the constructor
+and their possible overrides provided here.
+
+
+| Param | Type | Description |
+| --- | --- | --- |
+| options | [<code>HeaderGeneratorOptions</code>](#HeaderGeneratorOptions) | specifies options that should be overridden for this one call |
+
+
+* * *
+
+<a name="prepareHttpBrowserObject"></a>
+
+### `prepareHttpBrowserObject(httpBrowserString)`
+
+| Param | Type | Description |
+| --- | --- | --- |
+| httpBrowserString | <code>string</code> | a string containing the browser name, version and http version, such as "chrome/88.0.4324.182|2" |
+
+
+* * *
+
+<a name="prepareBrowserObject"></a>
+
+### `prepareBrowserObject(browserString)`
+
+| Param | Type | Description |
+| --- | --- | --- |
+| browserString | <code>string</code> | a string containing the browser name and version, such as "chrome/88.0.4324.182" |
+
+
+* * *
+
+<a name="Browser"></a>
+
+### `Browser`
+
+| Param | Type | Description |
+| --- | --- | --- |
+| name | <code>string</code> | One of "chrome", "firefox" and "safari". |
+| minVersion | <code>number</code> | Minimal version of browser used. |
+| maxVersion | <code>number</code> | Maximal version of browser used. |
+| httpVersion | <code>string</code> | Either 1 or 2. If none specified the global `httpVersion` is used. |
+
+
+* * *
+
+<a name="HeaderGeneratorOptions"></a>
+
+### `HeaderGeneratorOptions`
+
+| Param | Type | Description |
+| --- | --- | --- |
+| browsers | [<code>Array.&lt;Browser&gt;</code>](#Browser) | List of Browsers to generate the headers for. |
+| operatingSystems | <code>Array.&lt;string&gt;</code> | List of operating systems to generate the headers for.  The options are "windows", "macos", "linux", "android" and "ios". |
+| devices | <code>Array.&lt;string&gt;</code> | List of devices to generate the headers for. Options are "desktop" and "mobile". |
+| locales | <code>Array.&lt;string&gt;</code> | List of at most 10 languages to include in the `Accept-Language` request header. |
```

Comment:
examples of the locales, their structure and how to find them would be great

Reply:
I mean, "List of at most 10 languages" could also be [Czech, Slovak, English]

Reply:
I guess we can add smth like - "list of 10 Accept-Language compatible languages" and add link to the men https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language

Reply:
Will do.